portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About</title>
+    <link rel="stylesheet" href="about.css">
+    <script src="truths_and_lies.js"></script>
+  </head>
+  <body>
+    <div class="about_content">
+    <div id="header">
+        <a href="index.html"> <img src=/images/profile.jpg /> </a>
+        <div class="left">
+            <a href="#"> <p> About </p> </a>
+            <a href="survey.html"> <p> Survey </p> </a>
+            <a href="contact.html"> <p> Contact </p> </a>
+        </div>
+    </div>
+    <div id="about">
+        <h3>Want to get to know me? Let's play Two Truths One Lie</h3>
+        <div id="truth_and_lies_game">
+            <label id="question_num">Press Start to Begin!</label>
```

Comment:
A couple suggestions here:
For the id as I noted elsewhere, use a hyphen instead of underscore: question-num.

Next, the <label> tag is usually meant to describe an input that it is specifically paired with.  In this particular instance it looks like you want more of heading type piece of information.  Maybe an h4?

Reply:
Oh I almost forgot, I also wanted to say that the name "question-num" here is a bit misleading in the initial case, when there is no question number at all in the area but instead some text saying "Press Start to Begin!".  I'd suggest going in one of two ways:
1)  Changing the name to something like game-heading or game-status or something that indicates that this area contains some type of information about the status of the game, which makes sense for both the initial text prompt and the case when the question number displays instead.
2)  Alternatively, and this is probably what I would do if building a more robust app, I would have two separate elements.. one called game-start-prompt and one called question-number (see my long ramble in another comment about avoiding abbreviations in names haha).  And at the beginning of the game, have the question-number's display set to "none", and after the game is started, activate its display and turn the display of game-start-prompt to none instead.  That way you have two clearly defined elements, each with a specific purpose =)  But you don't need to necessarily get this in depth for a simple web page!

Reply:
Changed to game-status